flutter/example/lib/main.dart

```diff
@@ -338,3 +397,22 @@ Future<void> makeWebRequest(BuildContext context) async {
     },
   );
 }
+
+class ThemeProvider extends ChangeNotifier {
```

Comment:
we probably dont need this as its just part of the example for user feedback, keeping it simple make it easier for people to understand the important bits